src/gsi/gsi_files.cmake

```diff
@@ -522,6 +522,7 @@ rsearch.F90
 rtlnmc_version3.f90
 rtma_comp_fact10.f90
 satthin.F90
+screen_to_ncdiag.f90
```

Comment:
Same comment / question as before.  Shouldn't we move `screen_to_ncdiag.f90` to the `ncdiag` module?

Reply:
Addressed.